gcalcli/gcalcli.py

```diff
@@ -68,6 +68,8 @@
 from unicodedata import east_asian_width
 from collections import namedtuple
 
+from . utils import override_color_map, valid_override_colors
```

Comment:
We don't want to add relative imports.  You should be able to do absolute imports (even in py2) because of the import from __future__ above.